src/models/deprecated.ts

```diff
@@ -0,0 +1,14 @@
+import { SearchError } from './answersapi/SearchError';
+import { SearchRequest } from './core/SearchRequest';
+
+/**
+ * @deprecated AnswersError is deprecated and has been replaced by SearchError
```

Comment:
`@link SearchError` which I think should work

Reply:
Done!